pywbem/cim_operations.py

```diff
@@ -1131,6 +1152,22 @@ def last_reply_len(self):
         """
         return self._last_reply_len
 
+    @property
+    def conn_id(self):
+        """
+        :term:`connection id`:
+        String that is unique for each WBEMConnection object created across the
+        normal execution-life of the program using pywbem. The
```

Comment:
The second half of the first sentence: "... created across the normal execution-life of the program using pywbem" is confuding because it is not clear what is meant with "normal", and second because it is actually a statement about the value remaining the same. I suggest to remove that part of the sentence, because this is again explained better in the next sentence.

Reply:
Dropped that part of first sentence.